### DISCUSSION TITLE
ET =  ?

### DISCUSSION THREAD

**Discussion by ElectroLund** [MichaelUray/muTimer#9] [Q&A]
I'm not seeing a definition of the plots in the various graphs.  What does the ET plot represent?

**Comment by MichaelUray** [comment]
ET means the elapsed time, I just added it as description to the [readme](https://github.com/MichaelUray/muTimer/tree/main?tab=readme-ov-file#introduction).

This name gets used for IEC PLC timers as well.
https://www.plcacademy.com/plc-timers/
